wagtail_airtable/views.py

```diff
@@ -35,6 +36,10 @@ def post(self, request, *args, **kwargs):
         else:
             messages.add_message(request, messages.ERROR, "Could not import")
 
+        if (next_url := request.POST.get("next")) and url_has_allowed_host_and_scheme(
+            next_url, allowed_hosts=settings.ALLOWED_HOSTS, require_https=not settings.DEBUG
```

Comment:
👌🏼 for the `require_https` kwarg

having said that, any chance this can be covered by tests?

Reply:
Tests added in https://github.com/wagtail/wagtail-airtable/pull/56/commits/92b960d6b1d84bea33429741236cd92a27f6ef63